solidity/dashboard/src/hooks/useCurrentWidthLowerThanThreshold.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from "react"
+
+const useCurrentWidthLowerThanThreshold = (widthThreshold) => {
+  const [width, setWidth] = useState(window.innerWidth)
+
+  const handleWindowSizeChange = () => {
+    setWidth(window.innerWidth)
+  }
```

Comment:
The `handleWindowSizeChange` function should be rather inside the` useEffect` hook because a new instance of the `handleWindowSizeChange` is created on every render and `handleWindowSizeChange` function won't be removed in `window.removeEventListener("resize", handleWindowSizeChange)`. Thougts? 🤔